buildpacks/nodejs-corepack/buildpack.toml

```diff
@@ -2,7 +2,7 @@ api = "0.8"
 
 [buildpack]
 id = "heroku/nodejs-corepack"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Does https://github.com/heroku/buildpacks-nodejs/blob/main/buildpacks/nodejs-corepack/CHANGELOG.md need a `0.1.1` entry adding manually too?

Reply:
No, because `0.1.1` hasn't been released yet. It'll get released after I run the "Release Buildpack" action, at which point a PR will be opened to auto-append the version to the changelog.